var/spack/test_repos/spack_repo/integration_test/packages/smoke1/smoke1-src/add.cxx

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+
+int main(int argc, char ** argv){
+    if(argc != 3) {
+        std::cerr << "Improper number of arguments (" << argc-1 << "), 2 were expected\n";
+        return 1;
+    }
+    int a = atoi(argv[1]);
```

Comment:
This is not C++, this is C.

If you want to write C++, then import the relevant header.